modules/op-configuring-pipelines-as-code-logging.adoc

```diff
@@ -0,0 +1,151 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-logging_{context}"]
+= Configuring {pac} logging 
+
+You can configure {pac} logging by editing the `pac-config-logging` config map in the `TektonConfig` custom resource (CR).
+
+.Prerequisites
+
+* {pac} is installed on your cluster.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Administration* → *CustomResourceDefinitions*.
+
+. Use the *Search by name* box to search for the `tektonconfigs.operator.tekton.dev` custom resource definition (CRD) and click *TektonConfig* to see the CRD details page.
+
+. Click the *Instances* tab.
+
+. Click the *config* instance to see the `TektonConfig` CR details.
+
+. Click the *YAML* tab.
+
+. Edit the `loglevel.` fields under the `.options.configMaps.pac-config-logging.data` parameter based on your requirements.
++
+.Example: `TektonConfig` CR with {pac} log level fields set to `warn`
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  platforms:
+    openshift:
+      pipelinesAsCode:
+        options:
+          configMaps:
+            pac-config-logging:
+              data:
+                loglevel.pac-watcher: warn <1>
+                loglevel.pipelines-as-code-webhook: warn <2>
+                loglevel.pipelinesascode: warn <3>
+                zap-logger-config: |
+                  {
+                    "level": "info",
+                    "development": false,
+                    "sampling": {
+                      "initial": 100,
+                      "thereafter": 100
+                    },
+                    "outputPaths": ["stdout"],
+                    "errorOutputPaths": ["stderr"],
+                    "encoding": "json",
+                    "encoderConfig": {
+                      "timeKey": "ts",
+                      "levelKey": "level",
+                      "nameKey": "logger",
+                      "callerKey": "caller",
+                      "messageKey": "msg",
+                      "stacktraceKey": "stacktrace",
+                      "lineEnding": "",
+                      "levelEncoder": "",
+                      "timeEncoder": "iso8601",
+                      "durationEncoder": "",
+                      "callerEncoder": ""
+                    }
+                  }
+----
+<1> The log level for the `pipelines-as-code-watcher` component
+<2> The log level for the `pipelines-as-code-webhook` component
+<3> The log level for the `pipelines-as-code-controller` component
+
+. Optional: Create custom logging config map for the {pac} components by changing the `.env.value` for each component under `.options.deployments` field. The example below shows the configuration with the custom config map called `custom-pac-config-logging`.
```

Comment:
```suggestion
. Optional: Create custom logging config map for the {pac} components by changing the `.env.value` for each component under the `.options.deployments` field. The example below shows the configuration with the custom config map for `custom-pac-config-logging`.
```
I think "called" should be "for" here though I dont think there is anything against using "called" in the style guides.

Reply:
Because here I talk about the custom config map called `custom-pac-config-logging`, I think changing it to "for" changes the meaning of the sentence. With "for" it sounds like the example shows some `custom-pac-config-logging` object that contains custom config map, which is wrong, because `custom-pac-config-logging` is the config map.